rpcs3/Emu/Cell/lv2/sys_game.cpp

```diff
@@ -211,7 +211,14 @@ error_code _sys_game_watchdog_clear()
 
 u64 _sys_game_get_system_sw_version()
 {
-	return stof(utils::get_firmware_version()) * 10000;
+	try
```

Comment:
What are you trying to do here? Since exceptions are disabled, I think it will only result in std::terminate

Reply:
I think I implemented fmt::stringtofloat or something like that

Reply:
I worked around.
The emulator shouldn't be crashed simply due to a bad /dev_flash/vsh/etc/version.txt.

Reply:
Maybe try_to_float in StrUtil.h